mreg/api/v1/tests/tests.py

```diff
@@ -47,7 +48,7 @@ def get_token_client(self, username=None, superuser=True, adminuser=False):
 
     def add_user_to_groups(self, group_setting_name):
         groups = getattr(settings, group_setting_name, None)
-        if groups is None:
+        if groups is None:  # pragma: no cover (test QA)
```

Comment:
This pragma should be removed and instead create a test for it.

Reply:
Fixed in https://github.com/unioslo/mreg/pull/484/commits/3fb4213c07bbd979d25a4cf18a83dae61abdb22b